index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+  <title>Portfolio Website</title>
+</head>
+<body>
+    <div id="wrapper">
+        <header id="container">
+            <div>
+                <p class="logo"><a href="#">Suleiman</a></p>
+            </div>
+            <div id="navbar">
+                <a href="#"><img src="images/Hamburguer.png" alt="an image here" class="image"></a>
+            </div>
+        </header>
+        
+        <section>
+            <h1 id="name">I'm Suleiman Welcome To My Portfolio Website</h1>
+        
+             <p id="about">I'm a software developer and i can help you build a solution, implement a feature or build 
+                a website for your business, organization or firm as the case may be. feel free to connect
+                with me using the social media links.
+             </p>
+        
+        
+            <h3 class="connect"><a href="#">LET'S CONNECT</a></h3>
+        
+            <ul class="socialmedia">
+                <li><a href="#"><img src="images/twiter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/Linkedin.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/m.svg" alt="m"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/manito.svg" alt="magnito"></a></li>
+            </ul>
+        </section>
```

Comment:
As previous reviewer requested, there should same amount of margin with the Figma template. Kindly fix that.
Figma template:
<img width="530" alt="Screenshot 2022-12-02 at 3 20 40 AM" src="https://user-images.githubusercontent.com/25887253/205186563-2ef5ab72-a47c-40a9-bd4b-42251f52818e.png">

Your design:
![Screen Recording 2022-12-02 at 3 15 04 AM](https://user-images.githubusercontent.com/25887253/205186808-1423d6dd-2782-4756-875b-35be26e0fd0b.gif)
Also there is a overflowing issue horizontally on 375px. Kindly fix that. You can start with deleting 8px margin around body element.